core/src/main/java/org/akaza/openclinica/domain/rule/action/NotificationActionProcessor.java

```diff
@@ -363,10 +363,10 @@ public ParticipantDTO getParticipantInfo(StudySubject studySubject) {
 			pDTO.setParticipantId(studySubject.getLabel());
 
 			if(studySubject.getUserId()!=null) {
-				pDTO.setfName(studySubject.getStudySubjectDetail().getFirstName());
-				pDTO.setParticipantEmailAccount(studySubject.getStudySubjectDetail().getEmail());
-				pDTO.setPhone(studySubject.getStudySubjectDetail().getPhone());
-				pDTO.setIdentifier(studySubject.getStudySubjectDetail().getIdentifier());
+				pDTO.setfName(studySubject.getStudySubjectDetail()!=null? studySubject.getStudySubjectDetail().getFirstName():"");
```

Comment:
If study subject detail is null, we should not set these attributes since blank strings are not same as null values. I would do something like:
```
if (studySubject.getStudySubjectDetail() != null) {
   studySubject.setfName(studySubject.getStudySubjectDetail().getFirstName()); 
studySubject.setParticipantEmailAccount(studySubject.getStudySubjectDetail().getEmail());
   ...
   ...
}
```

Reply:
Done